### ISSUE TITLE
v3/master: ctl:ruleRemoveById invalid range

### ISSUE THREAD

**Issue by theMiddleBlue** [SpiderLabs/ModSecurity#2111]
Hi,

the secrule parser give me an error when the first argument of `ctl:ruleRemoveById` is a valid range:

- `ctl:ruleRemoveById:101-1999` -> OK
- `ctl:ruleRemoveById:200-1999` -> parse failed *Invalid range: 200-1999*

**regression test**
```json
[
  {
    "enabled":1,
    "version_min":300000,
    "title":"Testing ctl:ruleRemoveById - issue 1444",
    "expected":{
      "http_code":200
    },
    "client":{
      "ip":"127.0.0.1",
      "port":123
    },
    "request":{
      "headers":{
        "Host":"localhost",
        "User-Agent":"curl/7.38.0",
        "Accept":"*/*"
      },
      "uri":"index.php?foo=bar&z=xxx",
      "method":"GET",
      "body": ""
    },
    "server":{
      "ip":"127.0.0.1",
      "port":80
    },
    "rules":[
        "SecRuleEngine On",
        "SecRule ARGS:foo \"@rx ^bar$\" \"id:100,phase:1,ctl:ruleRemoveById=200-1999\"",
        "SecRule ARGS:z \"@rx ^xxx$\" \"id:1010,phase:1,deny,status:403\""
    ]
  }
]
```

**executing test**
```bash
# test/regression_tests test/test-cases/regression/issue-1444-2.json
ModSecurity 3.0.3 - tests
(options are not available -- missing GetOpt)

  # File Name                                         Test Name                                                             Passed?   
--- ---------                                         ---------                                                             -------   
  1 issue-1444-2.json                                 Testing ctl:ruleRemoveById - issue 1444                               failed!

Test failed. From: test/test-cases/regression/issue-1444-2.json.
Test name: Testing ctl:ruleRemoveById - issue 1444.
Reason: 
parse failed.
Rules error. File: issue-1444-2.json. Line: 2. Column: 27. Invalid range: 200-1999 

Ran a total of: 1 regression tests - 1 failed. 0 skipped test(s). 0 disabled test(s).
```

**Comment by airween** [comment]
PR sent #2113

**Comment by zimmerle** [comment]
Fixed as of cbd15ec1382f7e2c3e19a93ea9ca3e8b4ced44f6
